src/Library/demos/Revealer/main.js

```diff
@@ -0,0 +1,14 @@
+const button = workbench.builder.get_object("button");
+const revealer = workbench.builder.get_object("revealer");
+
+button.connect("clicked", () => {
```

Comment:
it's a toggle button, this isn't the right thing to listen to

check the toggle button library entry :)